docs/resources/rule_group.md

```diff
@@ -120,6 +120,7 @@ EOT
 
 ### Optional
 
+- `allow_editing_from_ui` (Boolean) Whether to allow editing of the rule group from the Grafana UI. Defaults to `false`.
```

Comment:
I am not sure I like this name because it allows editing in general (via file-provisioning, unstable API, which is used by UI, but can be accessed directly as well).

Reply:
I think it could just be disable_provenance and be described further in description. I was trying to be clever but it's not necessary 😃

Reply:
I think `disable_provenance` is better, as it would align more with the naming in the header and the references in the docs.

Reply:
Done!